CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## Unreleased
+
+**This is an update from rapier 0.19 to Rapier 0.21 which includes several stability improvements
+and new features. Please have a look at the
+[0.20 and 0.21 changelogs](https://github.com/dimforge/rapier/blob/master/CHANGELOG.md) of Rapier.**
+
+- Update to rapier `0.21`.
+- Update to nalgebra `0.33`.
+
 ## v0.27.0-rc.1 (18 June 2024)
 
```

Comment:
I think this should be merged into 1 as we havn't made a final release.

Mentionning both release candidates and final releases in changelog will only lead to misunderstandings